js/components/Message/BodyPieces.js

```diff
@@ -8,10 +8,15 @@ BodyPieces.propTypes = {
 function BodyPieces(props) {
   //@TODO add support for styles.
 
-  let pieces = props.pieces.map(function(piece) {
+  let pieces = props.pieces.map(function(piece, i) {
     const separator = " ";
+    // Body pieces don't ever change order so we can use the index
```

Comment:
Does that seem right @linclark, or do you know of another way to get around the warning?